src/platform/plugins/shared/console/server/lib/spec_definitions/js/retriever.ts

```diff
@@ -35,7 +35,9 @@ export const retriever = (specService: SpecDefinitionsService) => {
     },
     linear: {
       __template: {
-        retrievers: [{}],
+        query: 'search terms',
+        fields: ['field1', 'field2^2'],
```

Comment:
`field1` / `field2` will most likely never be fields in a mapping - and the user will end up deleting these all the time.
I would leave the template as it was before TBH and not add fields here and not add fields and query